jobs/build/client_sync/Jenkinsfile

```diff
@@ -73,40 +88,72 @@ node {
     commonlib.checkMock()
 
     def kind = params.KIND
-    def source_version = params.SOURCE_VERSION
 
-    currentBuild.displayName = "#${currentBuild.number} - ${kind} ${source_version}${(params.DRY_RUN) ? " [DRY_RUN]" : ""}"
+    currentBuild.displayName = "#${currentBuild.number} - ${kind} ${params.VERSION}${(params.DRY_RUN) ? " [DRY_RUN]" : ""}"
 
     stage("Validate params") {
-        if (!source_version) {
+        if (!params.VERSION) {
             error 'SOURCE_VERSION must be specified'
         }
-        dest_version = "${(kind in prefixes) ? prefixes[kind] : ""}${source_version}"
-        source_dir = params.OVERRIDE_LOCATION ? params.OVERRIDE_LOCATION : String.format(location[kind], source_version)
-        latest_dir = "latest/"
+        if (!params.LOCATION) {
+            error 'LOCATION must be specified'
+        }
+
+        dest_version = "${env.WORKSPACE}/${(kind in prefixes) ? prefixes[kind] : ""}${params.VERSION}"
```

Comment:
The value seems to be a path.

Reply:
Fixed in 964eee60af7e700f957d77678ab7e8635f588110